src/Nirum/Docs.hs

```diff
@@ -210,3 +242,30 @@ filterReferences (Image { imageTitle = t } : ix) = Text t : filterReferences ix
 filterReferences (Link { linkContents = children } : ix) =
     children ++ filterReferences ix
 filterReferences (i : ix) = i : filterReferences ix
+
+transformReferences :: (Url -> Url) -> Block -> Block
+transformReferences replace = \ case
```

Comment:
Is this lambda case? https://github.com/spoqa/nirum/pull/292/files#diff-0d34d0356e9cdc7c6ada1bb20a299daaR1